src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.2.0
+
+### New Feature
+
+- Support for self-signed and private CA with Docker
+- Buildkit port is now configurable (default 443)
+- Add option for sharing namespace with users within organization
+- UI pagination support for GitHub branches list
+- Add permanently user deletion command
+- Allow to scale down to zero nodes if no user pods are running
+- Populate oauth errors to the user
+- Added new helm values to set PSP and AppArmor
```

Comment:
The change is not about the helm values updates but more about the functionality. I see we now have `podSecurityPolicy.enabled=true` by default (with AppArmour enabled). Did this functionality change? If so we probably need to be a little more descriptive

Reply:
I've added a link to the docs. I've changed the description to express that this is now configurable